ui_framework/components/local_nav/_local_search.scss

```diff
@@ -44,6 +48,15 @@
     transform: translateY(-50%); /* 1 */
   }
 
+  .kuiLocalSearchAssistedInput__assistanceLink {
+    display: none;
+  }
+
+  .kuiLocalSearchInput:focus + .kuiLocalSearchAssistedInput__assistance .kuiLocalSearchAssistedInput__assistanceLink,
+  .kuiLocalSearchAssistedInput__assistanceLink:active {
+    display: inherit;
+  }
+
```

Comment:
That seems like a weird interaction to me. If the user wants to modify something at the end of their query they're going to mouse over it in order to click the spot they want to edit, but then that spot is going to suddenly disappear under the help link. The only way for them to get to what they wanted at that point is to click another part of the query and shift the cursor with the keyboard. I'd find that infuriating as a user.